Modified our team page to the current version of EvalAI

Sanji515 opened the pull request:
**Screenshot:**

![Screenshot from 2019-06-03 01-15-37](https://user-images.githubusercontent.com/32524438/58766527-54c3f980-859d-11e9-8157-78c7f0f0d121.png)

Sanji515 committed:
modified our team page

Shekharrajak commented on src/app/components/our-team/our-team.component.html:
```diff
@@ -1,24 +1,24 @@
-	<div class="our-team-title">
-		{{coreTeamType}}
-	</div>
+        <div class="our-team-title">
```
You can install some extension in  your editor to automatically make proper alignment when you save the file.

lunayach replied:
Yes, and if you use Py-Charm or VSCode, you could easily auto-format.

Sanji515 replied:
yeah I use VS code

Shekharrajak commented on src/app/components/our-team/our-team.component.scss:
```diff
@@ -87,22 +88,22 @@
       top: -40px;
       left: -50px;
-      background: $blue-overlay-dark;
+      background: #252833;;
```
Change the variable value `blue-overlay-dark`

Sanji515 replied:
@Shekharrajak done

Shekharrajak commented on src/app/components/our-team/our-team.component.scss:
```diff
@@ -150,8 +160,16 @@
       width: 100%;
       top: 0;
-      left: 0;
+      left: -15px;
```
Can we make it relative to the device width?

lunayach replied:
@Sanji515 is this done?

Sanji515 replied:
@lunayach I checked it but it was a very little change from the starting position of card so I keep it as it is and It's looking good in small devices also.

Sanji515 replied:
@lunayach @Shekharrajak done

Shekharrajak reviewed (commented):
Please make sure it works in different devices.

Sanji515 committed:
our-team-page

lunayach committed:
Merge branch 'master' into our-team-page

lunayach commented:
@Sanji515  Actually, I was unable to see it on the surge and not sure if it's working on all the devices. You may want to add tests and resolve a minor comment.

Sanji515 committed:
Add unit tests and made the suggested change relative to device

- Added unit tests for fetchOurTeamMembers method on init
- Made the top logo and personal website row relative to device

Sanji515 committed:
Merge branch 'our-team-page' of https://github.com/Sanji515/EvalAI-ngx into our-team-page

Sanji515 commented:
> You may want to add tests and resolve a minor comment

@lunayach  done. Please review

Sanji515 committed:
Merge branch 'master' into our-team-page

lunayach commented on src/app/components/our-team/our-team.component.scss:
```diff
@@ -140,18 +141,34 @@
 
   .team-link-head {
-      padding: 10px 15px;
+      padding: 10px 5px 10px 5px !important;
```
I see at many places you using `!important` (in other PR's as well). Generally, it's discouraged to use this and use only if we just can't do without this. Was this the case here?

Sanji515 replied:
@lunayach no, previously it was using `padding-left` from some other place, so just to override I used it but it is not required as I've removed that `padding-left`

Removed it now, please review

Sanji515 committed:
removed the unnecessary used important for applying the css in our team scss file

Sanji515 committed:
Merge branch 'our-team-page' of https://github.com/Sanji515/EvalAI-ngx into our-team-page

lunayach reviewed (approved):
I think, this could be merged.

Shekharrajak commented on src/app/components/our-team/our-team.component.html:
```diff
@@ -1,24 +1,24 @@
-                <a ng-if="data.personal_website !== 'null'" href="{{data.personal_website}}" target="_blank">Personal Website</a>
-            </span>
+                        <span class="col-md-1 col-sm-1 col-xs-1">
+                            <a ng-if="data.linkedin_url !== 'null'" href="{{data.linkedin_url}}" target="_blank"><i class="fab fa-linkedin"></i></a>
```
Isn't it still 80+ length. Also, HTML tags are not line by line.

Sanji515 replied:
@Shekharrajak done, please take a look if it looks good to you

Sanji515 committed:
formatted the our team component html

RishabhJain2018 committed:
Merge branch 'master' into our-team-page
